### PR TITLE
Video speedster

### PR DESCRIPTION
**Pull request by vdvibhu20**
speedcontrol

### CODE REVIEW

**Comment by vdvibhu20** [comment]
@abhishek97 @aayusharora

File: sass/styles/elements/elements.video-player.scss [hunk 1]
```diff
@@ -1,15 +1,16 @@
 .video-player{
-    @include dimensions(100%, 100%);
-    background: $black;
+  @include dimensions(100%, 100%);
```
Comment (aayusharora) [inline sass/styles/elements/elements.video-player.scss]: Please check why we need the height in percentages @vdvibhu20

File: sass/styles/objects/o-video-player/video-player.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    }
+  }
+
+  .select-box{
+    display: none;
+    @include border-radius(3px);
+    position: absolute;
+    bottom: 45px;
+    right: 15px;
+    width: 33px;
```
Comment (aayusharora) [inline sass/styles/objects/o-video-player/video-player.scss]: Does the select-box will maintain the constant width on all devices i.e 33px?

File: sass/styles/settings/settings.color.scss [hunk 3]
```diff
@@ -47,4 +47,4 @@ $otp-input-bg: #ebebeb;
 
 $hover-bg: #fafafa;
 $dropdown-divider: #f4f4f4;
-
+$video-speed: #262626;
```
Comment (aayusharora) [inline sass/styles/settings/settings.color.scss]: Please rename it :)

File: sass/styles/elements/elements.video-player.scss (hunk 1 above)
Comment (vdvibhu20) [reply sass/styles/elements/elements.video-player.scss]: yes that is needed as we want video player to take full dimensions of .content which in turn will cause <iframe> and <video> to take full dimensions.

File: sass/styles/objects/o-video-player/video-player.scss (hunk 2 above)
Comment (vdvibhu20) [reply sass/styles/objects/o-video-player/video-player.scss]: yes, less than that will cause overflow.

File: sass/styles/settings/settings.color.scss (hunk 3 above)
Comment (vdvibhu20) [reply sass/styles/settings/settings.color.scss]: renamed

**Comment by witty123** [comment]
@aayusharora lgtm, merge if changes are approved.

**Pull request merged**